content/docs/command-reference/plots/show.md

```diff
@@ -170,11 +170,11 @@ epoch,loss,accuracy
 Here's a corresponding `train.tsv` metrics file:
 
 ```
-epoch	loss	accuracy
-1	0.19	0.81
-2	0.11	0.89
-3	0.07	0.93
-4	0.04	0.96
+epoch loss accuracy
+1 0.19 0.81
+2 0.11 0.89
+3 0.07 0.93
+4 0.04 0.96
```

Comment:
Oops, looks like me running Prettier on this file replaced `\t` with a space in the TSV example. I'll fix that.